stories/core/Carousel.stories.tsx

```diff
@@ -105,3 +110,99 @@ Controlled.args = {};
 Controlled.argTypes = {
   activeSlideIndex: { control: { disable: true } },
 };
+
+const useId = () => 'my-custom-carousel';
+export const Custom: Story<CarouselProps> = () => {
+  const gradients = [
+    { from: '#cc2b5e', to: '#753a88' },
+    { from: '#00467f', to: '#a5cc82' },
+    { from: '#2193b0', to: '#6dd5ed' },
+    { from: '#ffe000', to: '#799f0c' },
+    { from: '#e65c00', to: '#f9d423' },
+    { from: '#1488cc', to: '#2b32b2' },
+    { from: '#bbd2c5', to: '#536976' },
+    { from: '#9796f0', to: '#fbc7d4' },
+    { from: '#b79891', to: '#94716b' },
+    { from: '#acb6e5', to: '#86fde8' },
+  ];
+
+  const id = useId();
+  const [current, setCurrent] = React.useState(0);
+
+  return (
+    <section
+      aria-roledescription='carousel'
+      tabIndex={0}
+      style={{ display: 'inline-grid', width: 'min(90vw, 40vh)' }}
+    >
+      <ol
+        style={{
+          listStyle: 'none',
+          margin: 0,
+          padding: 0,
+          display: 'grid',
+          grid: `[slide] 1fr / [slide] 1fr`,
+        }}
+      >
+        {gradients.map(({ from, to }, index) => (
+          <li
+            key={index}
+            role='tabpanel'
+            id={`${id}-slide-${index}`}
+            onClick={({ currentTarget: { clientWidth }, clientX }) => {
+              const diff = clientWidth - clientX > clientWidth / 2 ? -1 : +1;
+              setCurrent(
+                (prev) => (gradients.length + prev + diff) % gradients.length,
+              );
+            }}
+            style={{
+              gridArea: 'slide',
+              opacity: current === index ? 1 : 0,
+              pointerEvents: current === index ? 'auto' : 'none',
+              transition: 'opacity 0.5s',
+              cursor: 'pointer',
+            }}
+          >
+            <div
+              style={{
+                background: `linear-gradient(to right, ${from}, ${to})`,
+                height: 'min(600px, 70vh)',
+                display: 'grid',
+                placeItems: 'center',
+                fontSize: 48,
+                color: 'hsl(0deg 0% 100% / 0.7)',
+              }}
+            >
+              {index}
```

Comment:
Maybe start from `1` as in other stories? :D

Reply:
done